pydreamer/envs/wrappers.py

```diff
@@ -25,8 +25,9 @@ def step(self, action):
         obs, reward, done, info = self.env.step(action)  # type: ignore
         self.step_ += 1
         if self.step_ >= self.time_limit:
-            done = True
             info['time_limit'] = True
+            info['terminal'] = done
```

Comment:
Not sure if we want this, because the environment might set `done=True` because of time limit, not because of actual terminal state. For example, CartPole might have internal time limit of 200, after which it returns done. We want to wrap it in `TimeLimitWrapper(time_limit=200)`, which changes the last step to `time_limit=True`.

Reply:
Oh, that's nasty, two bugs cancel each other out :)

Fun tangent: reading the cartpole code I see a comment that the env terminates in 500 steps (used to be 200), but I don't see the termination itself. Probably some magic happens when you register.